docker/rocal-on-ubuntu-20.dockerfile

```diff
@@ -17,7 +17,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install initramfs-tools libnuma-de
         wget ${ROCM_INSTALLER_REPO} && \
         sudo apt-get install -y ./${ROCM_INSTALLER_PACKAGE} && \
         sudo apt-get update -y && \
-        sudo amdgpu-install -y --usecase=graphics,rocm --no-32
+        sudo amdgpu-install -y --usecase=rocm
```

Comment:
please add these to the other 2 dockerfiles as well: pytorch  & tensorflow.
Also needs change to the rhel file